2_process.R

```diff
@@ -129,6 +133,8 @@ p2_targets_list <- list(
     proportion_lc_by_prms(p2_NLCD_LC_w_catchment_area %>%
                             # filtering to only the comid_downs of each PRMS - nrow = ~459
                             filter(comid %in% p2_drb_comids_down$comid),
+                          area_col = 'TOTDASQKM',
```

Comment:
for NLCD ACC variables, do we want to take the area of the catchment (AREASQKM) col or the Total upstream area (TOTDAKM) ?